File: 测试界面/Main/ViewControllers/ViewController.m
//
//  ViewController.m
//  测试界面
//
//  Created by shoule on 15/5/20.
//  Copyright (c) 2015年 SaiDicaprio. All rights reserved.
//

#import "ViewController.h"
#import "CartViewController.h"
@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)MyCart {
    CartViewController *next = [CartViewController new];
    [self.navigationController pushViewController:next animated:YES];
}

@end

File: 大众AutoLayout题目/ViewController.swift
//
//  ViewController.swift
//  大众AutoLayout题目
//
//  Created by shoule on 16/4/12.
//  Copyright © 2016年 SaiDicaprio. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet weak var midView:UIView?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }
    
    @IBAction func TapClick(sender:AnyObject){
        midView?.hidden = true
        print("123")
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

